### PR TITLE
ODH workbench: Build the runtime images on post and pre submit jobs

### PR DESCRIPTION
**Pull request by harshad16**
Related-to: https://github.com/opendatahub-io/notebooks/issues/54
Depends-on: https://github.com/opendatahub-io/notebooks/pull/63

This would build the runtime image and push them to quay repo.

### CODE REVIEW

**Review by atheo89 — CHANGES_REQUESTED** [review]

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml [hunk 1]
```diff
@@ -424,6 +506,61 @@ tests:
...
+  skip_if_only_changed: ^docs/|\\.(md)$|^(\.gitignore|LICENSE|OWNERS|PROJECT)$
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: runtime-cuda-tensorflow-ubi9-python-3.9
+    env:
+      IMAGE_REPO: workbench-images
+      RELEASE_VERSION: runtime-cuda-tensorflow-ubi9-python-3.9
+    workflow: opendatahub-io-ci-image-mirror
+
 - as: notebooks-e2e-tests
```
[the remark this answers is not in the document]
Comment (harshad16) [reply ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml:559]: Included tests for the runtimes as well :+1:

**Pull request merged**